main/src/ca/uwaterloo/flix/language/phase/optimizer/OccurrenceAnalyzer.scala

```diff
@@ -161,6 +161,18 @@ object OccurrenceAnalyzer {
           (Expr.IfThenElse(e1, e2, e3, tpe, eff, loc), ctx4)
         }
 
+      case Expr.While(exp1, exp2, tpe, eff, loc) =>
+        val (e1, ctx1) = visitExp(exp1)
+        val (e2, ctx2) = visitExp(exp2)
+        // TODO: No clue what to do here
+        //       Maybe OnceInLambda is kind of the right thing?
```

Comment:
@jaschdoc when you have time, do you know what we should do here about the occurance of the while body?

Reply:
I think we need a new Occur variant for this

Reply:
you can think of while as being `while(() -> cond)(() -> body)`, doesnt that justify the lambda occurance?

Reply:
Yes :)